File: SmartCtlDriveWebApp/ClientApp/src/app/models/drive-information.ts
export class DriveInformation {
    id: number;
    serialNumber: string;
    modelName: string;
    modelFamily: string;
    formFactor: string;
    blockCount: number;
    bytesCount: number;
    firmwareVersion: string;
    rpm: number;
    ataVersionMinor: number;
    ataVersionMajor: number;
    ataVersionName: string;
    sataSpeed: string;
    smartOK: boolean;
    hasErrorRecovery: boolean;
    powerOnHours: number;
    powerCycleCount: number;
    temperature: number;
}

File: SmartCtl.Domain/DriveInformation.cs
using System;
using System.Collections.Generic;
using System.Text;
using SmartCtlDriveApp.Model;

namespace SmartCtl.Domain
{
    public class DriveInformation
    {
        public int ID { get; set; }
        public string SerialNumber { get; set; }
        public string ModelName { get; set; }
        public string ModelFamily { get; set; }
        public string FormFactor { get; set; }
        public long BlockCount { get; set; }
        public long BytesCount { get; set; }
        public string FirmwareVersion { get; set; }
        public int RPM { get; set; }
        public int ATAVersionMinor { get; set; }
        public int ATAVersionMajor { get; set; }
        public string ATAVersionName { get; set; }
        public string SATASpeed { get; set; }
        public bool SmartOK { get; set; }
        public bool HasErrorRecovery { get; set; }
        public int PowerOnHours { get; set; }
        public int PowerCycleCount { get; set; }
        public int Temperature { get; set; }
    }
}
